### DISCUSSION TITLE
DH icon remains colorless / DH cannot be started

### DISCUSSION THREAD

**Discussion by tubefan** [aclap-dev/video-downloadhelper#1452] [Help and Q&A]
### Your question

Since today the DH icon remains colorless and I can't start DH. What can I do?

### Addon & CoApp version and details

⊖
Video DownloadHelper BETA 9.0.1.7a1 (beta)
Video DownloadHelper BETA
Version 9.0.1.7a1 (beta)
Browsersprache: de
Produktionsversion
Kompilierungszeitpunkt: 2024-05-08
Kompilierungsoptionen: browser=firefox, linuxlic=false, noyt=false, target=mozilla
⊖
Plattform
Plattform: Win x86-64
Browser: Mozilla Firefox 132.0.1
⊖
Begleitanwendung installiert
Begleitanwendung gefunden: VdhCoApp 2.0.20
Pfad zur Begleitanwendung: C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You have the latest Firefox & the latest CoApp.  But your beta version of VDH itself is woefully out of date.  You should upgrade to the latest version (9.1.0.41)

After you install the current VDH beta, if you're still having problems, post at least one URL of a video that isn't working for you.  Don't post the name of a web site.  I am not going to scrounge around on some web site I've never heard of looking for examples.  Give me at least one example.  Post the URL of a web page on which a video resides so I can test it.

**Reply by tubefan** [reply]
Thank you very much - it seems to work!
